### DISCUSSION TITLE
Is it possible to get raw text from container directive?

### DISCUSSION THREAD

**Discussion by podlomar** [remarkjs/remark#779] [Q&A]
I am using the remark-directive plugin. I would like to have a piece of YAML inside a container directive such as
```
:::config
some: yaml
with: values
:::
```

But the plugin returns the inside of the container already parsed to markdown. I suppose I could use the position in the AST to extract the raw unparsed content, but it doesn't feel right. Is there a better way?

**Comment by wooorm** [comment]
See: https://github.com/micromark/micromark-extension-directive/issues/11#issuecomment-880446409.
